doc(release) Release notes web 24.04.22 ot 24.04.06 (#4978)
centreon/centreon-documentation: versioned_docs/version-24.04/releases/centreon-os.mdx

## Centreon Web

### 24.04.22

Release date: `January 19, 2026`

<details open>
  <summary>Enhancements</summary>

- [Custom Views] Optimized performance when retrieving ACLs in widgets.

</details>

<details>
<summary>Bug fixes</summary>

> - [CLAPI] Fixed an issu where the APPLYTPL command can take a long time to take effect.

Reviewer: ```suggestion
- [CLAPI] Fixed an issue with the APPLYTPL command taking a long time to take effect.
```

> - [Dashboards] Resource Table widget - Fixed link to Resource Status page

Reviewer: ```suggestion
- [Dashboards] Resource Table widget - Fixed link to the Resource Status page
```

> - [Users] Fixed issue preventing user from being created when using an autologin key.

Reviewer: ```suggestion
- [Users] Fixed an issue preventing users from being created when an autologin key was used.
```

</details>

<details>
<summary>Security fixes</summary>

- [Security] fixed a vulnerability in Administration > Parameters > Images.
- [Security] Fixed a vulnerability in LDAP configuration.

> - [Security] fixed a vulnerability in Monitoring > Performance > Virtual Metrics.

Reviewer: ```suggestion
- [Security] Fixed a vulnerability in Images parameters.
- [Security] Fixed a vulnerability in LDAP configuration.
- [Security] Fixed a vulnerability in Virtual Metrics.
```

> - [Security] fixed a vulnerability in Notification > Hostgroups.

Reviewer: ```suggestion
- [Security] Fixed a vulnerability in Hostgroups notifications.
```

## Centreon Open Tickets

### 24.04.6

Release date: `January 19, 2026`

<details>
  <summary>Security fixes</summary>

> - [BREAKING CHANGE] [Open Tickets] The ability to define a custom command linked to a rule has been removed, due to low adoption and security concerns.

Reviewer: ```suggestion
- [BREAKING CHANGE] [Open Tickets] Defining a custom command linked to a rule is no longer possible due to security concerns.
```
